services/traefik-forward-auth-mgmt/0.3.9/defaults/cm.yaml

```diff
@@ -34,11 +34,6 @@ data:
       whitelist: []
       enableRBAC: true
       enableImpersonation: true
-      rbacPassThroughPaths:
```

Comment:
was just looking back in history to try to understand why this was added in the first place and if we want to be removing this since it was explicitly added in the past.. most I can find is https://github.com/mesosphere/kubernetes-base-addons/pull/815/commits/346aede1c7cd9270d11f6e53171e616ff976966f where kiali was added because "skip tfa rbac as kiali handles it" (commit message). Just want to make sure we actually want to undo this (not sure what was the impetus in the first place), let's check with product (and deepak authored this commit in the past so check with him again, maybe he can recall why 😄 )

Reply:
true. Maybe keep Kiali there but only removing kube dashboard?

Reply:
talked to Deepak, kiali needs to stay